CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# 19.0.0
+
+- Upgrade to typescript v5
+- **BREAKING:** Use `tspc` instead of `ttsc`
```

Comment:
Fwiw - there are other breaking changes for users. `prettier` v3 has breaking changes that users will likely experience and so does Jest v29.

Additionally, any team that is using Lerna v6, will need to upgrade to Lerna v7 (or switch to nx). Lerna v6 is incompatible with Prettier v2. 

I just went through this same exercise in the integration SDK project.